src/api/services/usersService.ts

```diff
@@ -78,5 +79,5 @@ export const enrichWithUserFromParams = async (
 };
 
 export const getAllUserFromParticipant = async (participant: Participant) => {
-  return participant!.$relatedQuery('users').castTo<User[]>();
+  return participant!.$relatedQuery('users').where('deleted', 0).castTo<User[]>();
```

Comment:
I think we remove the association between participant and user when we delete a user, is this still necessary?

Reply:
It's not necessary.  I removed it.